proto/desmos/profiles/v1beta1/models_chain_links.proto

```diff
@@ -93,4 +93,18 @@ message Base58Address {
 
   // Value contains the Base58-encoded address
   string value = 1 [ (gogoproto.moretags) = "yaml:\"value\"" ];
+}
+
+// EthAddress represents an Eth address
+message EthAddress {
```

Comment:
I think we should name this something like `HexAddress` since the other ones have names that are related to the way they are encoded (`Bech32Address`, `Base58Address`) and not the chains they are used on (`CosmosAddress`, `SolanaAddress`).

Reply:
Maybe we should call it `EthHexAddress`.

Reply:
I would go for a generically `HexAddress` too in order to be consistent with the names we used for the other chains.

Reply:
I noticed that the eth address applies `keccak256` to hash uncompressed public key into address while solana and cosmos utilizes `ripemd160` hasher and compressed public key. 
Any other chains might use different algorithm to generate `base58/bech32` address as well, like `Tron` uses `keccak256` with uncompressed public key then encode the address byte into base58, the current `Base58Address` will not be valid to `Tron`.

Reply:
@dadamu I think it's better to open an issue on this specific finding so that we can see how we should edit the `AddressData` interface to support different encoding as well

Reply:
@RiccardoM 
Okay, I will rename it into hex address then note that current `HexAddress` mainly supports eth address.

Reply:
@dadamu No need to note that it supports ETH addresses specifically. We can just specify that it supports keccak256-compressed addresses

Reply:
@RiccardoM @bragaz Updated.